astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -166,26 +165,19 @@ def test_future_altaz():
     if hasattr(utils, '__warningregistry__'):
         utils.__warningregistry__.clear()
 
-    with catch_warnings() as found_warnings:
-
-        location = EarthLocation(lat=0*u.deg, lon=0*u.deg)
-        t = Time('J2161')
-
-        SkyCoord(1*u.deg, 2*u.deg).transform_to(AltAz(location=location, obstime=t))
+    location = EarthLocation(lat=0*u.deg, lon=0*u.deg)
+    t = Time('J2161')
```

Comment:
Note: It is good practice to move things that are not expected to emit warning outside of the context manager.